### PR TITLE
Create fine-focus.csl

### PR DESCRIPTION
**Pull request by jenniferhuddleston**

### CODE REVIEW

File: fine-focus.csl [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+    <id>http://www.zotero.org/styles/fine-focus</id>
+    <link href="http://www.zotero.org/styles/fine-focus" rel="self"/>
+    <link href="http://www.zotero.org/styles/annual-reviews" rel="template"/>
+    <link href="http://finefocus.org/submissions" rel="documentation"/>
+    <author>
+      <name>Jennifer Huddleston</name>
+      <email>jrh11a@acu.edu</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="generic-base"/>
```
Comment (rmzelle) [inline fine-focus.csl]: Can you change this to `<category field="biology"/>`?

Comment (jenniferhuddleston) [inline fine-focus.csl]: Yes. What is the change it? I changed it in my original file. Where do I need to resubmit that file?

Comment (rmzelle) [inline fine-focus.csl]: Use the pencil/edit icon at https://github.com/jenniferhuddleston/styles/blob/patch-1/fine-focus.csl

Comment (jenniferhuddleston) [inline fine-focus.csl]: done

**Comment by rmzelle** [comment]
Great, thanks!

**Pull request merged**